dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/DataManager.kt

```diff
@@ -241,7 +251,7 @@ class DataManager(
     fun getDataSet(dataId: String, dataType: DataType, correlationId: String): StorableDataSet {
         assertActualAndExpectedDataTypeForIdMatch(dataId, dataType, correlationId)
         val dataMetaInformation = metaDataManager.getDataMetaInformationByDataId(dataId)
-        val dataAsString = getDataFromStorage(dataId, correlationId)
+        val dataAsString = getDataFromStorageLocation(dataId, correlationId)
```

Comment:
Can we improve naming?
like: 
`getDataFromStorage` -> `getDataFromStorageService`
`getDataFromStorageLocation` -> `getDataFromCacheOrStorageService`

I am happy for better suggestions - however `getDataFromStorageLocation` doesn't make me expect what the method actually does.

Reply:
👍

Reply:
Done

Reply:
ok